### DISCUSSION TITLE
AWS X-Ray integration

### DISCUSSION THREAD

**Discussion by stefanoschrs** [ory/keto#444] [Ideas]
Hello, is it somewhere down the pipeline to integrate [AWS X-Ray](https://aws.amazon.com/xray/) for tracing?

**Comment by vinckr** [comment]
As far as I know we have no concrete plans as of now, but we are always open for feature requests. 
We can use this discussion thread to see if there are others in the community who would like to see this feature!

**Reply by emayssat** [reply]
why would authorization integrate with tracing? and how? 
If requests are blocked, they will report an error status which should be observable in X-Ray, correct?

**Reply by aeneasr** [reply]
@vinckr this was recently merged by a community member and is supported now: https://github.com/ory/x/blob/master/tracing/tracer.go#L131 (unless I'm confusing something)

**Reply by stefanoschrs** [reply]
Hey @aeneasr Elastic Application Performance Monitoring (APM) is a different thing.
